en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-config.md

```diff
@@ -0,0 +1,1774 @@
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            Yes
+        </td>
+    </tr>
+</table>
+
+**Sample Configuration of URI configs**
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<localEntry key="uriConnection" xmlns="http://ws.apache.org/ns/synapse">
+    <mongodb.init>
+        <name>uriConnection</name>
+        <connectionType>URI</connectionType>
+        <connectionURI>mongodb+srv://server.example.com/?connectTimeoutMS=300000&authSource=aDifferentAuthDB</connectionURI>
+        <database>users</database>
+    </mongodb.init>
+</localEntry>
+```
+
+### Common configs for STANDARD and DSL types
+
+<table>
+    <tr>
+        <th>Parameter Name</th>
+        <th>Type</th>
+        <th>Description</th>
+        <th>Default Value</th>
+        <th>Required</th>
+    </tr>
+    <tr>
+        <td>
+            Replica Set
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Specifies the name of the replica set, if the mongod is a member of a replica set.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Auth Source
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Specify the database name associated with the user’s credentials. 
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Auth Mechanism
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Specify the authentication mechanism that MongoDB will use to authenticate the connection.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Auth Mechanism Properties
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Specify properties for the specified authMechanism as a comma-separated list of colon-separated key-value pairs.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Gssapi Service Name
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Set the Kerberos service name when connecting to Kerberized MongoDB instances. This value must match the service name set on MongoDB instances to which you are connecting.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            W
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Corresponds to the write concern w Option. The w option requests acknowledgement that the write operation has propagated to a specified number of mongod instances or to mongod instances with specified tags.
```

Comment:
'mongod' or 'mongoDB'?
Shall we rephrase this sentence: "Corresponds to the write concern w Option."
Not sure what this means

Reply:
```suggestion
Corresponds to the write concern w Option.
```